src/Uno.UI/UI/Xaml/Controls/RichEditBox/RichEditBox.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Windows.UI.Xaml.Controls
 {
-	public partial class RichEditBox : Control
+	public partial class RichEditBox // : Control
```

Comment:
!

Reply:
This is a workaround for #10724, I'll remove.